cmd/bee/cmd/split_test.go

```diff
@@ -60,3 +64,72 @@ func TestDBSplit(t *testing.T) {
 		t.Fatalf("got %d hashes, want %d", gotHashes, wantHashes)
 	}
 }
+
+func TestDBSplitChunks(t *testing.T) {
+	t.Parallel()
+
+	s := (rand.Intn(10) + 10) * 1024 // rand between 10 and 20 KB
+	buf := make([]byte, s)
+	_, err := crand.Read(buf)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	inputFileName := path.Join(t.TempDir(), "input")
+	err = os.WriteFile(inputFileName, buf, 0644)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	dir := path.Join(t.TempDir(), "chunks")
+	err = os.Mkdir(dir, os.ModePerm)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	err = newCommand(t, cmd.WithArgs("split", "chunks", "--input-file", inputFileName, "--output-dir", dir, "--r-level", "3")).Execute()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	stat, err := os.Stat(inputFileName)
+	if err != nil {
+		t.Fatal(err)
+	}
+	want := api.CalculateNumberOfChunks(stat.Size(), false)
+
+	entries, err := os.ReadDir(dir)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if int64(len(entries)) < want {
+		t.Fatalf("want at least %d chunks", want)
+	}
+
+	for _, entry := range entries {
```

Comment:
what you want to test is, do the split of the file yourself and compare that to what is stored